recipes/assertpy/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: assertpy
+  version: !!str 0.12
+
+source:
+  fn: assertpy-0.12.tar.gz
+  url: https://pypi.python.org/packages/54/66/1ed435aa2f51e39d73ff4c59dc9bc569278fcaf9586599891b146a8f9e5a/assertpy-0.12.tar.gz#md5=e0d382ca6e3788e564f0ec6c24162c37
+  md5: e0d382ca6e3788e564f0ec6c24162c37
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+
+test:
+  imports:
+    - assertpy
+
+about:
+  home: https://github.com/ActivisionGameScience/assertpy
+  license: BSD 3-Clause
```

Comment:
Please manually package the `license_file` as described in the below link. Then raise an issue upstream in the github repo about including it in the source distribution, adding the link to the issue in a comment in this recipe above the `license_file` entry:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually